src/pseudopeople/configuration/interface.py

```diff
@@ -0,0 +1,52 @@
+"""
+===========================
+The Configuration Interface
+===========================
+
+An interface for user's to interact with the configuration (config) in pseudopeople.
+
+In ``pseudopeople`` a configuration is used to provide noise level that get passed to the noise functions to apply
+various types of noise to given datasets.  Users can provide their own configuration that will override the deault
+values.
```

Comment:
I think we should list the allowable dataset names here

Reply:
Or maybe as part of the docstring for the dataset_name arg in the function itself? Maybe that's better

Reply:
I like the second option.

Reply:
Ok I can add that.  This is what I was saying earlier.  Wanting to provide that list by linking it or just the full list itself.